code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -639,12 +639,13 @@
 		"Basic" = "starman",
 	)
 	speed_modifier = CYBORG_STARMAN_SPEED_MODIFIER
+	default_modules = 0
```

Comment:
null. 0 isn't a list.

Reply:
I'm 99% sure this is a boolean that dictates whether the flash and flashlight are added to robot stuff.

Reply:
I am mixing bleeding edge with my WiP branch. Att least change this to FALSE for consistency.